daemon/cmd/hubble.go

```diff
@@ -88,6 +89,8 @@ func (d *Daemon) getHubbleStatus(ctx context.Context) *models.HubbleStatus {
 }
 
 func (d *Daemon) launchHubble() {
+	ctx, cancel := context.WithCancel(d.ctx)
+	defer cancel()
```

Comment:
`launchHubble` is not blocking, so If we use this new derived context and defer its cancellation, it will be cancelled as soon as `launchHubble` returns.
I don't think that is what we want, looking at how `ctx` is used in `namespaceManager.Run()` and in the two `select` blocks at lines 284 and 311.